testBaseUrl.js

```diff
@@ -1,11 +1,11 @@
 define([], function() {
```

Comment:
This file could probably be excluded from linting, but it's so small it's no problem fixing it :smile: